input.go

```diff
@@ -27,6 +32,30 @@ func (t *Terminal) TypedKey(e *fyne.KeyEvent) {
 		}
 	case fyne.KeyTab:
 		_, _ = t.in.Write([]byte{'\t'})
+	case fyne.KeyF1:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', 'O', 'P'})
+	case fyne.KeyF2:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', 'O', 'Q'})
+	case fyne.KeyF3:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', 'O', 'R'})
+	case fyne.KeyF4:
+		_, _ = t.in.Write([]byte{asciiEscape, '[', 'O', 'S'})
```

Comment:
F1 through F4 don't have `[` in them.

```suggestion
		_, _ = t.in.Write([]byte{asciiEscape, 'O', 'P'})
	case fyne.KeyF2:
		_, _ = t.in.Write([]byte{asciiEscape, 'O', 'Q'})
	case fyne.KeyF3:
		_, _ = t.in.Write([]byte{asciiEscape, 'O', 'R'})
	case fyne.KeyF4:
		_, _ = t.in.Write([]byte{asciiEscape, 'O', 'S'})
```